core/editor/src/document/document_message_handler.rs

```diff
@@ -662,6 +663,24 @@ impl MessageHandler<DocumentMessage, &InputPreprocessor> for DocumentMessageHand
 					responses.push_back(DocumentOperation::SetLayerTransform { path, transform }.into());
 				}
 			}
+			SetLayerCoordinates(path, x_option, y_option) => {
```

Comment:
Do you think "Position" or "Location" would be less ambiguous or more standardized compared to "Coordinates"?

Reply:
Good point, going with "Translation" as it matches the transform terms